examples/docs_snippets/docs_snippets/guides/dagster/reexecution/workspace.yaml

```diff
@@ -0,0 +1,2 @@
+load_from:
+  - python_file: pipeline/unreliable_pipeline.py
```

Comment:
nit: if we don't refer the content to this code snippet or expect users to run the code using the workspace.yaml, we don't need this file. - im just trying to make it as simple as possible, but if you find it necessary or helpful, feel free to ignore my comment.

Reply:
Yeah, I was testing the pipeline by running `dagit` within the `reexecution` folder, and seemed to get an error if I didn't have the workspace.yaml file. For the future, happy to discuss a better way to test the pipelines locally!

Reply:
I see. That makes sense. Alternatively, you can do `dagit -f <py filename>`.
IMO, this is good to keep and we can maybe refer users to this folder and say "you can simply run `dagit` in that folder to follow the example` or something.

Reply:
Ok, will keep!